src/conditional-flow/boolean-conditions.js

```diff
@@ -4,7 +4,8 @@
 function getResult (didPass) {
 
   // TODO: write code in this function body to pass the tests
-
+if (true) alert('Well done, you passed!')
```

Comment:
the function is asking to check if the value that has been passed to the getResult function is true or false, and return a different string.

```
if(didPass === true) {
   return "Well done, you passed!"
}
```